### ISSUE TITLE
how can I finetune docvqa on another languange?

### ISSUE THREAD

**Issue by Mousaic** [google-research/pix2struct#37]
I wanna use the pretrained docvqa model to continue finetuning.I only change the question to another language,and the labels keep the same,can this solution work out?

**Comment by sl5035** [comment]
Hi,
I am also currently in a similar situation. Have you figured it out?

**Comment by chopin1998** [comment]
any update on this?
